apps/demo/src/index.scss

```diff
@@ -64,7 +65,7 @@ html {
 .container {
   margin: auto;
   position: relative;
-  height: 100%;
+  overflow: hidden;
```

Comment:
This is currently clipping the pattern image background. But as discussed previously, we should not use a `img` tag but css instead to display this decorative background, this should fix this clipping issue.

<img width="541" alt="image" src="https://github.com/openfun/cunningham/assets/9265241/7ac16d04-4ffe-4f3e-bde0-ea302b012590">